pkg/server/server.go

```diff
@@ -384,6 +393,9 @@ func (h *handler) doMapping(identity *token.Identity) (string, []string, error)
 			if err != nil {
 				return "", nil, fmt.Errorf("mapper %s renderTemplates error: %v", m.Name(), err)
 			}
+			if m.UsernamePrefixEnforce() && ReservedPrefixExists(username, m.UsernamePrefixReserveList()) {
+				return "", nil, fmt.Errorf("mapper %s username %s has reserved prefix: %v", m.Name(), username, m.UsernamePrefixReserveList())
```

Comment:
Can we change the text to something more informative? Right now, a user trying to debug why their user is getting authorization errors won't understand this message.

How about something along the lines of:
```
invalid username '%s' for mapper %s: username must not begin with with the following prefixes: %v